mlflow/xgboost.py

```diff
@@ -48,6 +49,13 @@
     ENSURE_AUTOLOGGING_ENABLED_TEXT,
     batch_metrics_logger,
 )
+
+# Pylint doesn't detect objects used in class keyword arguments (e.g., metaclass) and considers
+# `ExceptionSafeAbstractClass` as 'unused-import': https://github.com/PyCQA/pylint/issues/1630
+from mlflow.utils.autologging_utils import (  # pylint: disable=unused-import
+    ExceptionSafeAbstractClass,
+)
+
```

Comment:
First, I tried this:

```python
from mlflow.utils.autologging_utils import ( 
    ...
    exception_safe_function,
    ExceptionSafeAbstractClass,  # pylint: disable=unused-import
    ...
)
```

but didn't work. Then, I tired this:

```python
from mlflow.utils.autologging_utils import ExceptionSafeAbstractClass,  # pylint: disable=unused-import
```

`black` converts this to:

```python
from mlflow.utils.autologging_utils import (
    ExceptionSafeAbstractClass,
)  # pylint: disable=unused-import
```

This didn't work either. Then, I settled in the current code.